psychopy/experiment/components/apriltag/__init__.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Part of the PsychoPy library
+# Copyright (C) 2002-2018 Jonathan Peirce (C) 2019-2021 Open Science Tools Ltd.
+# Distributed under the terms of the GNU General Public License (GPL).
+
+from __future__ import absolute_import, print_function
+
+from contextlib import contextmanager
+from os import path
+from pathlib import Path
+from psychopy.experiment.components import image
+from psychopy.experiment.components.image import ImageComponent, Param, getInitVals
+from psychopy.localization import _translate
+
+
+class AprilTagComponent(ImageComponent):
```

Comment:
I'm not sure we need a component for this - if I'm interpreting it correct, all this does is assemble the path of a specific image file from a set, then presents it in an image component? Adding a component would put it in the components panel for everyone - I'm not sure this is a good idea for such a specific use case.

Instead, why not add a function to the library which is something like this:
```suggestion
def getAprilTagImage(family, id, root=Path(__file__).parent):
    """
    Get the path to an AprilTag image from its family and id.
    """
    # Map family tags to image prefices
    _apriltag_family_name_to_image_prefix = {
        "tag16h5": "tag16_05_",
        "tag25h9": "tag25_09_",
        "tag36h11": "tag36_11_",
        "tagCircle21h7": "tag21_07_",
        "tagCircle49h12": "tag49_12_",
        "tagCustom48h12": "tag48_12_",
        "tagStandard41h12": "tag41_12_",
        "tagStandard52h13": "tag52_13_",
    }
    # Make sure we have a valid tag name
    assert family in _apriltag_family_name_to_image_prefix, _translate(f"Invalid family name, should be one of: {list(_apriltag_family_name_to_image_prefix)}")
    # Make sure root is a path
    root = Path(root)
    # Construct path to image
    return str(
        root + _apriltag_family_name_to_image_prefix[family] + str(id).zfill(5)
    )
```

You could put it in its own folder (maybe within plugins? or maybe in the iohub sub-module?) alongside the images, then just instruct users to set their Image components' values to:
```
$plugins.apriltag.getAprilTagImage(desiredFamily, desiredId)
```
(or wherever the function ends up being)

That way we don't clutter up the components panel with hyper-specific components, while retaining an easy way to use the AprilTag stimuli fresh out of the box.